src/Stores/NodesStore.js

```diff
@@ -397,7 +397,7 @@ class NodesStore extends BaseStore {
     }
 
     onSetModuleConf(nodeId, name, configuration, tag, cb) {
-        const query = `set_module_conf('${name}', ${configuration ? `'${configuration}'` : 'nil'});`;
+        const query = `set_module_conf('${name}', ${configuration ? `${configuration}` : 'nil'});`;
```

Comment:
It would be better to use query arguments instead of "building' the query, thus:

```
query = set_module_conf(name, configuration);
```
and as arguments:
```
{
  "name": name,
  "configuration: configuration
}
```

This way, it is also fine if configuration is `nil`, since that will be correctly packed.

Reply:
This will be changed in a later release (~v0.4.4).